CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ else()
         -Wno-deprecated-register
         -Wno-mismatched-new-delete
         -Wno-deprecated-declarations
+        -Wno-unused-value
```

Comment:
Generally it is bad to have unused values, what is the warning? Can we try to fix that?

Reply:
```
[build] [ 85%] Building CXX object demo_example/CMakeFiles/CountChar.dir/CountChar.cpp.o
[build] async_simple/demo_example/CountChar.cpp: In function ‘async_simple::coro::Lazy<int> CountTextChar(const Texts&, char)’:
[build] async_simple/demo_example/CountChar.cpp:50:1: error: statement has no effect [-Werror=unused-value]
[build]    50 | }
[build]       | ^
[build] At global scope:
```

```c++
Lazy<int> CountTextChar(const Texts &Content, char c) {
    int Result = 0;
    for (const auto &line : Content)
        Result += co_await CountLineChar(line, c);
    co_return Result;
}
```

I think it's a bug of gcc10.

Reply:
Agreed. But disable `-Wno-unused-value` looks over acted. I feel we can add a `(void)Result;` before `co_return` to eliminate the warning.